src/System.Drawing.Primitives/src/System/Drawing/PointF.cs

```diff
@@ -0,0 +1,208 @@
+        /// </devdoc>
+        public static PointF operator -(PointF pt, SizeF sz)
+        {
+            return Subtract(pt, sz);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Compares two <see cref='System.Drawing.PointF'/> objects. The result specifies
+        ///       whether the values of the <see cref='System.Drawing.PointF.X'/> and <see cref='System.Drawing.PointF.Y'/> properties of the two <see cref='System.Drawing.PointF'/>
+        ///       objects are equal.
+        ///    </para>
+        /// </devdoc>
+        public static bool operator ==(PointF left, PointF right)
+        {
+            return left.X == right.X && left.Y == right.Y;
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Compares two <see cref='System.Drawing.PointF'/> objects. The result specifies whether the values
+        ///       of the <see cref='System.Drawing.PointF.X'/> or <see cref='System.Drawing.PointF.Y'/> properties of the two
+        ///    <see cref='System.Drawing.PointF'/> 
+        ///    objects are unequal.
+        /// </para>
+        /// </devdoc>
+        public static bool operator !=(PointF left, PointF right)
+        {
+            return !(left == right);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Translates a <see cref='System.Drawing.PointF'/> by a given <see cref='System.Drawing.Size'/> .
+        ///    </para>
+        /// </devdoc>
+        public static PointF Add(PointF pt, Size sz)
+        {
+            return new PointF(pt.X + sz.Width, pt.Y + sz.Height);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Translates a <see cref='System.Drawing.PointF'/> by the negative of a given <see cref='System.Drawing.Size'/> .
+        ///    </para>
+        /// </devdoc>
+        public static PointF Subtract(PointF pt, Size sz)
+        {
+            return new PointF(pt.X - sz.Width, pt.Y - sz.Height);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Translates a <see cref='System.Drawing.PointF'/> by a given <see cref='System.Drawing.SizeF'/> .
+        ///    </para>
+        /// </devdoc>
+        public static PointF Add(PointF pt, SizeF sz)
+        {
+            return new PointF(pt.X + sz.Width, pt.Y + sz.Height);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Translates a <see cref='System.Drawing.PointF'/> by the negative of a given <see cref='System.Drawing.SizeF'/> .
+        ///    </para>
+        /// </devdoc>
+        public static PointF Subtract(PointF pt, SizeF sz)
+        {
+            return new PointF(pt.X - sz.Width, pt.Y - sz.Height);
+        }
+
+        public override bool Equals(object obj)
+        {
+            if (!(obj is PointF)) return false;
+            PointF comp = (PointF)obj;
+            return
+            comp.X == X &&
+            comp.Y == Y &&
+            comp.GetType().Equals(GetType());
```

Comment:
The comment about value types not being derivable from may be a clue; it suggests someone was over cautious, and then later someone removed a check like this, adding the comment as explanation as to why it was safe, but they didn't do the equivalent here.